src/ui/React/LogBoxManager.tsx

```diff
@@ -398,7 +398,20 @@ function LogWindow({ hidden, script, onClose }: LogWindowProps): React.ReactElem
               <div style={{ display: "flex", flexDirection: "column" }}>
                 {script.logs.map(
                   (line: React.ReactNode, i: number): React.ReactNode =>
-                    typeof line !== "string" ? line : <ANSIITypography key={i} text={line} color={lineColor(line)} />,
+                    typeof line !== "string" ? (
+                      line
+                    ) : (
+                      <ANSIITypography
+                        key={i}
+                        text={line}
+                        color={lineColor(line)}
+                        styles={{
+                          display: "inline-block",
+                          fontSize: Settings.styles.tailFontSize,
+                          lineHeight: `calc(${Settings.styles.tailFontSize}px * ${Settings.styles.lineHeight})`,
```

Comment:
This didn't need to happen before; why is it needed now?

Reply:
The inline block is needed to make the line height match what was set by the user for smaller and bigger font sizes than the default (see the images in my comment).
The line height might not be needed since the set value should cascade down. I'll play around a bit and remove it, if it isn't necessary.